.github/dependabot.yml

```diff
@@ -4,12 +4,16 @@ updates:
       directory: '/'
       schedule:
           interval: 'monthly'
-          day: 'monday'

```

Comment:
These ended up not impacting the monthly schedule, which always executes on the 1st of the month. So I took them out.